maven-compat/src/main/java/org/apache/maven/profiles/DefaultMavenProfilesBuilder.java

```diff
@@ -48,11 +47,9 @@ public ProfilesRoot buildProfiles(File basedir) throws IOException, XmlPullParse
 
         if (profilesXml.exists()) {
```

Comment:
This method suffers from encoding problems and appears to not be used.  I'd rather just return null for this method and remove the dependency on plexus-utils and the encoding problems altogether.